django/contrib/auth/backends.py

```diff
@@ -90,9 +96,15 @@ def has_module_perms(self, user_obj, app_label):
     def get_user(self, user_id):
         UserModel = get_user_model()
         try:
-            return UserModel._default_manager.get(pk=user_id)
+            user = UserModel._default_manager.get(pk=user_id)
         except UserModel.DoesNotExist:
             return None
+        if user:
+            is_active = getattr(user, 'is_active', None)
+            if (is_active or (is_active is False and self.allow_inactive_users) or
```

Comment:
I think it's not - it might be `False` or `None` and we return user either account is inactive and backend allows it, or account does not have this field. What do you think?